platform-sdk/swirlds-platform-core/src/main/java/com/swirlds/platform/state/MerkleStateRoot.java

```diff
@@ -1021,6 +1030,21 @@ public PlatformStateModifier getWritablePlatformState() {
         return writablePlatformStateStore();
     }
 
+    /**
+     * {@inheritDoc}
+     */
+    @NonNull
+    @Override
+    public RosterStateModifier getRosterStateModifier() {
+        if (isImmutable()) {
+            throw new IllegalStateException("Cannot get writable roster store when state is immutable");
+        }
```

Comment:
These lines can be replaced with `throwIfImmutable()`. This is not consistent throughout this class, so not a required change. In the future though, we should try to replace all of these with the one line method. Feel free to close with no action.

Reply:
Sure. If there are further changes to make in this PR, I'll include this too.

Reply:
done